spec/vcr_cassettes/kubeconfig/cluster_not_ready.yml

```diff
@@ -57,7 +57,7 @@ http_interactions:
     uri: http://127.0.0.1:8000/
```

Comment:
These are VCR cassettes that reproduce a previous recorded HTTP session. Unfortunately I couldn't find a way for them to support a TLS connection. Since this is for testing purposes I'd suggest revisit it during july or any other time.